cmd/nsed/main.go

```diff
@@ -1,54 +1,113 @@
 package main
 
 import (
-	"encoding/json"
+	"context"
 	"flag"
-	"log"
 	"os"
-	"os/signal"
-	"syscall"
+	"strings"
+	"time"
 
+	"github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc"
+	"github.com/ligato/vpp-agent/api/configurator"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/networkservice"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/registry"
+	"github.com/networkservicemesh/networkservicemesh/pkg/tools"
+	"github.com/networkservicemesh/networkservicemesh/sdk/common"
+	"github.com/networkservicemesh/networkservicemesh/sdk/endpoint"
+	"github.com/opentracing/opentracing-go"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
 	"gopkg.in/yaml.v3"
 
+	"vl3nse/internal/vl3"
+	"vl3nse/internal/vppagent"
 	"vl3nse/pkg/nseconfig"
 )
 
 var (
-	configFile = flag.String("cfg", "/etc/cnns-nse/config.yaml", "the path to config file, default: /etc/cnns-nse/config.yaml")
+	configFile = flag.String("config", "/etc/cnns-nse/config.yaml", "the path to config file, default: /etc/cnns-nse/config.yaml")
 
-	logger = log.New(os.Stderr, "test", 0)
+	nsPodIp          = envOrDefault("NSE_POD_IP", "2.2.20.0")
```

Comment:
Note: 
We already have envOrDefault implementation in github.com/dtornow/cnns-nsr/nsrenv package. The repo is private and I can understand why it is not easy to import it directly...